source/api_documentation/index.rst

```diff
@@ -29,6 +29,9 @@ used to ingest calls from external sources for conversation intelligence process
 :doc:`calls_in_progress_api/index` -
 used to query/update calls that are currently in progress within the Invoca platform
 
+:doc:`sms_messaging_api/index` -
+used to enable/disable promo numbers for SMS, as well as send SMS messages
```

Comment:
Given the [MVP doc product features description](https://invoca.atlassian.net/wiki/spaces/DEV/pages/59409170433/SMS+MVP+Source+of+Truth+Doc#Project-Milestones-And-Product-Expectations) and maybe the [SMS alpha epic description](https://invoca.atlassian.net/browse/STORY-11414), how about we make this a bit more broad? Maybe:

`used for SMS number management, as well as handling incoming/outgoing SMS messages within the Invoca platform`

Would be good to get feedback from @jbuehring

Reply:
Yeah, I like that. 👍

Reply:
:shipit: